lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+{
+  "guest": "Ospite",
+  "browse": "Esplora",
+  "search": "Cerca",
+  "library": "Libreria",
+  "lyrics": "Testi",
+  "settings": "Impostazioni",
+  "genre_categories_filter": "Filtra categorie o generi...",
+  "genre": "Genere",
+  "personalized": "Personalizzato",
+  "featured": "Featured",
+  "new_releases": "Nuove Release",
+  "songs": "Canzoni",
+  "playing_track": "In riproduzione {track}",
+  "queue_clear_alert": "Questo svuoterà la coda attuale. {track_length} brani saranno rimossi\nVuoi continuare?",
+  "load_more": "Carica altro",
+  "playlists": "Playlist",
+  "artists": "Artisti",
+  "albums": "Album",
+  "tracks": "Brani",
+  "downloads": "Download",
+  "filter_playlists": "Filtra le tue playlist...",
+  "liked_tracks": "Brani Preferiti",
+  "liked_tracks_description": "Tutti i tuoi brani preferiti",
+  "create_playlist": "Crea Playlist",
+  "create_a_playlist": "Crea una playlist",
+  "create": "Crea",
+  "cancel": "Annulla",
+  "playlist_name": "Nome Playlist",
+  "name_of_playlist": "Nome della playlist",
+  "description": "Descrizione",
+  "public": "Pubblico",
+  "collaborative": "Collaborative",
+  "search_local_tracks": "Cerca tra i brani locali...",
+  "play": "Riproduci",
+  "delete": "Elimina",
+  "none": "Nessuno",
+  "sort_a_z": "Ordina da A-Z",
+  "sort_z_a": "Ordina da Z-A",
+  "sort_artist": "Ordina per Artista",
+  "sort_album": "Ordina per Album",
+  "sort_tracks": "Ordina Brani",
+  "currently_downloading": "Attualmente in Downloading ({tracks_length})",
+  "cancel_all": "Annulla Tutto",
+  "filter_artist": "Filtra artisti...",
+  "followers": "{followers} Follower",
+  "add_artist_to_blacklist": "Aggiungi artista a blacklist",
+  "top_tracks": "Migliori Brani",
+  "fans_also_like": "Ai Fans piace anche",
+  "loading": "Caricamento...",
+  "artist": "Artista",
+  "blacklisted": "Blacklisted",
+  "following": "Following",
+  "follow": "Follow",
```

Comment:
@OpenCode should we translate it with "Seguiti" and "Segui" ?

Reply:
Personally I don't like when "Follow" or "Followers" is translated. But "Segui" for "Follow" sounds good.